packages/p2p/src/components/my-profile/my-profile-stats/my-profile-balance/my-profile-balance.jsx

```diff
@@ -2,14 +2,12 @@ import * as React from 'react';
 import { Money, Text } from '@deriv/components';
 import { isMobile } from '@deriv/shared';
 import { observer } from 'mobx-react-lite';
-import { useUpdatingAvailableBalance } from 'Components/hooks';
 import { Localize } from 'Components/i18next';
 import { useStores } from 'Stores';
 
 const MyProfileBalance = () => {
     const { general_store, my_profile_store } = useStores();
     // const [is_balance_tooltip_open, setIsBalanceTooltipOpen] = React.useState(false);
```

Comment:
is this commented out for a reason? if so could we add a TODO for this or remove it

Reply:
There's a TODO in the file already.